azkaban-common/src/main/java/azkaban/utils/Emailer.java

```diff
@@ -78,12 +78,12 @@ public Emailer(final Props props, final CommonMetrics commonMetrics) {
 
     if (props.getBoolean("jetty.use.ssl", true)) {
       this.scheme = HTTPS;
-      this.clientPortNumber = Integer.toString(props.getInt("jetty.ssl.port",
-          Constants.DEFAULT_SSL_PORT_NUMBER));
+      this.clientPortNumber = Integer.toString(props.getInt("mail.jetty.ssl.port", props.getInt("jetty.ssl.port",
```

Comment:
Please define the new key names in the Constant class. This will make them easier to discover.